cassandra-analytics-core/src/main/java/org/apache/cassandra/spark/validation/StartupValidatable.java

```diff
@@ -0,0 +1,6 @@
+package org.apache.cassandra.spark.validation;
+
+public interface StartupValidatable
```

Comment:
Add detailed java docs on interfaces

Reply:
Added class-level JavaDocs to all newly added files.